src/sbin/sympa.pl.in

```diff
@@ -212,10 +212,17 @@ if ($main::options{'upgrade_config_location'}) {
             $Conf::Conf{'etc'};
     }
 
+    ## Check if db_type is not the boilerplate one
+    if ($Conf::Conf{'db_type'} eq '(You must define this parameter)') {
+        die sprintf
```

Comment:
I feel It'd better to be like
```
    ...
    unless ($Conf::Conf{'db_type'} and $Conf::Conf{'db_type'} =~ /\A\w+\z/) {
        die sprintf
    ...
```

Reply:
I added this check in 0807b7972e4d9506fbc70bdddb7723cbefcd5a63, but left the first one: this check does not allow us to be sure it’s the boilerplate `db_type`, so the `die` message has to be different.
Is it ok for you?

Reply:
I'm ok for your change. Thanks!